gems/smithy/lib/smithy/templates/client/waiters.erb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# This is generated code!
+
+require 'smithy-client/waiters'
+
+module <%= module_name %>
+
+  # @api private
+  module Waiters
+<% waiters.each do |waiter| -%>
+<% if waiter.deprecated -%>
+    # @deprecated
+<% end -%>
+    # @api private
+    #
+    # <%= waiter.documentation %>
```

Comment:
You'll need to loop over this. Check out other examples in the package. The documentation may have newlines in it. I believe the other methods are called docstrings and split on newline. Just make it consistent.